src/ext/default.c

```diff
@@ -83,40 +104,51 @@ static double TFIDFNormDocLenScorer(const ScoringFunctionArgs *ctx, const RSInde
 
 /* recursively calculate score for each token, summing up sub tokens */
 static double bm25Recursive(const ScoringFunctionArgs *ctx, const RSIndexResult *r,
```

Comment:
Shouldn't the `explain` go into the args?

Reply:
tfidfRecursive doesn't get the ctx but now that I'm looking at it, I added a 'bool explain' everywhere else instead of adding ctx only there.
We discussed using it.
It can be set at result_processor.c:167 and then used later.

Reply:
Once again, we thought of additional future flags.
Whatever you prefer is fine with me.

Reply:
For now we can even check the flag directly since it will be 0 otherwise

Reply:
Not sure if this prior to the latest changes